src/pages/Register.css

```diff
@@ -0,0 +1,39 @@
+.box1{
+    display: grid;
+    place-items: center;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 10%;
+}
+.child{
+    border: 1px solid black;
+    display: grid;
+    place-items: center;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 3vh;
```

Comment:
you should use rem for padding, and no more than 4rem: if you need more, you're facing a layout problem, not spacing. try flexbox/grid on the container instead